dangerfile.js

```diff
@@ -0,0 +1,32 @@
+/* global danger, fail, message */
+import { message, danger } from 'danger';
+
+const validChangesetCategories = ['Added', 'Changed', 'Removed', 'Fixed'];
+
+// Check for correct Changeset formatting
+danger.git.created_files.filter((filepath) => filepath.includes('.changeset/'))
```

Comment:
Might also be worth executing this as part of a Git hook just to save people having to do another commit.

We can still have it in this file though on the off-chance someone has disabled husky hooks :)

Reply:
Yep – happy to make a ticket for this as an engineering improvement to-do.

It's probably worth testing it, as it does take a little while to run when I've been testing it, so it could get a bit annoying if it makes valid commits hang every time.